README.md

```diff
@@ -1,12 +1,21 @@
-#This project was abandoned because [Moxie Marlinspike said he was not OK with LibreSignal using the Open Whisper Systems servers and the name "Signal"](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165).
+#This GitHub repository has been abandoned.
```

Comment:
I don't remember if dates are exact, but apart from that this timeline is correct.